tests/unit/test_collector.py

```diff
@@ -41,3 +44,182 @@ def test_01_mega_raid_collector_installed_and_okay(self):
         self.assertEqual(len(list(payloads)), len(available_metrics))
         for payload in payloads:
             self.assertIn(payload.name, available_metrics)
+
+    def test_10_lsi_sas_2_collector_not_installed(self):
+        """Test LSI SAS 2 collector when sas2ircu is not installed."""
+        lsi_sas_2_collector = LSISASControllerCollector(2)
+        payloads = lsi_sas_2_collector.collect()
```

Comment:
The response here should be mock?
I know the there won't be response if the command is not installed. But it's relate on the user's local environment. Right?

Reply:
Yes